ktor-client/ktor-client-webrtc/ktor-client-webrtc-rs/common/src/io/ktor/client/webrtc/rs/Utils.kt

```
@@ -0,0 +1,104 @@
+
+internal fun Stats.toKtor() = WebRtc.Stats(
+    id = id,
+    type = type,
+    timestamp = timestamp.toLong(),
+    props = mapOf("native-stats" to props)
+)
+
+internal suspend inline fun withSdpException(crossinline block: suspend () -> Unit) {
+    try {
+        block()
+    } catch (e: RtcException.SdpException) {
```

detekt (1.23.8)

[warning] The caught exception is swallowed. The original exception could be lost.

(detekt.exceptions.SwallowedException)